tools/profile_apply

```diff
@@ -250,12 +250,23 @@ def apply(tabs):
         for chart_rows in split_by_title(rows):
             put_chart(chart_rows)
 
+    def clear_form(form_id):
+        """Removes all the form_fields from the given form."""
+        db.execute('update form_field set parent_form_field = null'
+                   ' where form_id = %s', form_id)  # remove foreign keys
+        db.execute('delete from form_field where form_id = %s', form_id)
+
     def apply_chart(rows, form_id):
         """Applies the chart definition given rows from the chart tab."""
+        clear_form(form_id)
         section = None
         sections = []
         section_rows = []
+
         for row in rows:
+            if row['title'].strip():
+                sections.append(('[chart_divider]',
```

Comment:
So, if I understand this correctly:
- there's still only one `chart` section in the `profile.csv`
- each entry in the `title` adds a new `chart_divider` to the chart form in OpenMRS
- When this is parsed in the client, the chart form is split up based on the `chart_divider`s?